src/js/components/modals/ServiceScaleFormModal.js

```diff
@@ -58,7 +97,7 @@ class ServiceGroupFormModal extends mixin(StoreMixin) {
         min: 0,
         name: 'instances',
         placeholder: instancesCount,
-        value: instancesCount,
+        value: instancesCount.toString(),
```

Comment:
As it ends up inside an `input` field it will eventually become one, so I guess yes. But ideally the Form should take care of that for us.